stacks_queues/stack.py

```diff
@@ -12,27 +12,42 @@ def push(self, element):
         """ Adds an element to the top of the Stack.
             Returns None
         """
-        pass
+        self.store.add_first(element)
+        
+        return None
```

Comment:
👍 

And no need to explicitly `return None` here. Saying the function returns None is the same as not returning anything. I only explicitly `return None` if there are other places in the function where I am returning some other value (if one path through a function returns, then all paths through the function should return).